app/models/grda_warehouse/data_source.rb

```diff
@@ -37,7 +37,7 @@ class GrdaWarehouse::DataSource < GrdaWarehouseBase
   accepts_nested_attributes_for :projects
 
   scope :importable, -> do
-    source.where(authoritative: false)
+    source.where(authoritative: false, hmis: nil)
   end
```

Comment:
What do you think of this solution? we use the scope in lots of places.
I tested out an even with an HMIS source when you mark it as non-authoritative, you can still import HUD ZIPs in the UI, so thats good since we need that for migrating. (So I guess the importable scope isn't used in determining whether an import is allowed).

Reply:
That works for me!  Thanks.